README.md

```diff
@@ -38,6 +38,7 @@ Possible parameters:
 
 Possible parameters:
 * `grid` - (true|false) Defaults to false. Set to true to enable the alternative grid view
+* `layout` - (1|2|3) Defaults to 1. 1) 4 columns, 2) 3 columns, 3) simplified 3 columns
```

Comment:
hey @rafaeldalpra i'm testing this here: http://pcplayground.flywheelsites.com/fundraisers/fundraisers-grid/ and the first block of fundraisers is layout=1  looks like its pulling the wrong layout.